contrib/forms/body_composition/new.php

```diff
@@ -156,16 +141,16 @@ function rbinput($name, $value, $desc, $colname)
  <tr>
   <td width='5%' nowrap>Body Type</td>
   <td colspan='2' nowrap>
-    <?php echo rbinput('form_body_type', 'Standard', 'Standard', 'body_type') ?>&nbsp;
-    <?php echo rbinput('form_body_type', 'Athletic', 'Athletic', 'body_type') ?>&nbsp;
+    <?php echo text('form_body_type', 'Standard', 'Standard', 'body_type') ?>&nbsp;
+    <?php echo text('form_body_type', 'Athletic', 'Athletic', 'body_type') ?>&nbsp;
```

Comment:
Note rbinput is a function, so revert changes in above 2 lines and change the rbinput function to:
```
function rbinput($name, $value, $desc, $colname)
{
    global $row;
    $ret  = "<input type='radio' name='" . attr($name) . "' value='" . attr($value) . "'";
    if ($row[$colname] == $value) {
        $ret .= " checked";
    }
    $ret .= " />" . text($desc);
    return $ret;
}
```